pandas/io/clipboard.py

```diff
@@ -83,8 +83,12 @@ def to_clipboard(obj, excel=None, sep=None, **kwargs):  # pragma: no cover
             if sep is None:
                 sep = '\t'
             buf = StringIO()
-            obj.to_csv(buf, sep=sep, **kwargs)
-            clipboard_set(buf.getvalue())
+            # clipboard_set (pyperclip) expects unicode
+            obj.to_csv(buf, sep=sep, encoding='utf-8', **kwargs)
```

Comment:
so you still should pass ``encoding='utf-8'`` to ``.to_csv``; the issue is that ``.to_clipboard()`` takes ``kwargs`` which *could* have ``encoding`` as one of them. So you need to explicity check if its invalid.